README.md

```diff
@@ -30,85 +29,79 @@ $ pip install sentinelhub --upgrade
+  * authentication and rate-limit handling,
 
-### AWS data download
+- geospatial utilities
+  * interface for geospatial objects and transformations,
+  * large area splitting,
+  * data collection objects,
+  * IO tools,
 
-The package allows to download Sentinel-2 data from Sentinel-2 on AWS
-and reconstruct data into ESA .SAFE format.
+- download Sentinel-2 data from public [AWS S3 buckets](https://registry.opendata.aws/sentinel-2/)
+  * restoration of .SAFE format,
+  * L1C and L2A data,
+  * command line interface,
 
-The following are implemented:
- * support for Sentinel-2 L1C and Sentinel-2 L2A data;
- * support of old and new (i.e. compact) .SAFE format;
- * support for downloading of either entire product, or a map of the .SAFE
- structure only;
- * support of command lines entries;
- * adjustable threaded downloads, and optional redownloads of existing data (not default);
- * requires either S-2 product ID, or tile name and date of a product.
+- [Geopedia](http://portal.geopedia.world/) WMS and REST API.
 
-In case of Sentinel-2 L2A data AWS access key is required.
 
-
-### Documentation
+## Documentation
 
 For more information on the package and to access the documentation, visit [readthedocs](http://sentinelhub-py.readthedocs.io/).
 
 
-# Examples
+## Examples
+
+The package has a collection of Jupyter notebooks with examples. They are available in the [examples folder](https://github.com/sentinel-hub/sentinelhub-py/tree/master/examples) on GitHub and converted into documentation under [Examples section](https://sentinelhub-py.readthedocs.io/en/latest/examples.html).
 
-Jupyter notebooks on how to use the modules to execute OGC requests, or
-download raw data from AWS in .SAFE format can be found in the [examples](https://github.com/sentinel-hub/sentinelhub-py/tree/master/examples/)
-folder, or viewed in the [docs](http://sentinelhub-py.readthedocs.io/):
- * Using OGC web services ([link](http://sentinelhub-py.readthedocs.io/en/latest/examples/ogc_request.html))
- * Using utilities for large geographical areas ([link](http://sentinelhub-py.readthedocs.io/en/latest/examples/large_area_utilities.html))
- * AWS data download ([link](http://sentinelhub-py.readthedocs.io/en/latest/examples/aws_request.html))
- * AWS data download from command line in .SAFE format ([link](http://sentinelhub-py.readthedocs.io/en/latest/aws_cli.html))
+Additionally, some examples are explained in Sentinel Hub webinar videos:
 
-# Blog posts
+- [Process API in Python](https://www.youtube.com/watch?v=sX3w3Wd3FBw&list=PL46vEE2ks3tn8NGesSFllgJW5MSYRi4od&index=10&t=2220s)
+- [OGC API in Python](https://www.youtube.com/watch?v=CBIlTOl2po4&list=PL46vEE2ks3tn8NGesSFllgJW5MSYRi4od&index=4&t=1766s)
+
+
+## Blog posts
+
+The package has been used in the background of many projects and use cases described at [Sentinel Hub blog](https://medium.com/sentinel-hub). The following blog posts are about the package itself:
```

Comment:
The package **has been used in the background of** many projects and use cases described at
-> The package **played a key role in** many projects and use cases described at

Reply:
fixed